conf/test_base.conf

```diff
@@ -282,6 +282,10 @@ lshw {
   lshw.defaultNicCapacity=10000000000
 }
 
+gpu {
+  gpuVendors = ["NVIDIA Corporation"]
```

Comment:
This is a stutter. `gpu.gpuVendors`? Maybe `gpu.supportedVendorStrings` or something to be more explicit